package.json

```diff
@@ -49,8 +49,8 @@
     "lint:scripts": "cd scripts && substrate-dev-run-lint"
   },
   "dependencies": {
-    "@polkadot/api": "^5.8.4-1",
-    "@polkadot/apps-config": "^0.95.1",
+    "@polkadot/api": "^5.9.1",
```

Comment:
The resolution is set to an older version still, but 5.9.1 is released now so the dependency updated. Is this OK or should this be bought down to match the resolution?

Reply:
Looks alright to me. I might be wrong but isn’t it pretty risk-free to bump the resolutions too?

Reply:
Basically, the resolutions are copied from https://github.com/polkadot-js/apps/blob/master/package.json, so I'm a little hesitant to deviate I guess. That said, everything seems to have decided on 5.9.1 regardless, so the resolution isn't really having an impact:

```
 % yarn why @polkadot/api
├─ @polkadot/api-derive@npm:5.9.1
│  └─ @polkadot/api@npm:5.9.1 (via npm:^5.8.4-4)
│
└─ @substrate/api-sidecar@workspace:.
   └─ @polkadot/api@npm:5.9.1 (via npm:^5.8.4-4)
```

I guess I'd be inclined to leave it as is I guess.